src/articles/ArticlePreview.sc.js

```diff
@@ -78,33 +85,42 @@ const Title = styled.div`
 `;
 
 const Difficulty = styled.div`
-  padding: 0 0.55em;
-  width: 2.8em;
-  height: 3.8em;
-  text-align: center;
-  border-radius: 50%;
-  background-color: ${veryLightGrey};
   display: flex;
-  align-items: center;
-  align-self: center;
-  justify-content: center;
-  flex-shrink: 0;
-  float: right;
+  align-items: flex-end;
+  flex-direction: row;
+  gap: 0.5em;
+  margin-right: 2em;
+  img {
+    height: 1.5em;
+  }
+  span {
+    font-weight: 450;
+  }
 `;
 
-const WordCount = styled(Difficulty)`
-  background-color: ${veryLightGrey};
-  align-self: center;
+const WordCount = styled(Difficulty)``;
```

Comment:
I like. So although they're the same, you're defining a specific name, that we can decide later to overwrite if we need to!